### PR TITLE
Fix error in packing and publish

### PR DESCRIPTION
**Pull request by ali-tayebi**
This PR fixes the issue in adding proto files to the project

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,17 +1,35 @@
...
+pack-build:
 	docker build -t protobuf-nuget-packer ./pack
 
-.PHONY: run-pack
-run-pack:
+.PHONY: pack-run
+pack-run:
 	docker run --rm -it --name protobuf-nuget-packer \
-		--volume $(PWD)/protosx:/protos --volume $(PWD)/artifacts:/artifacts\
+		--volume $(PWD)/../certificate-generation-contracts/protos:/protos --volume $(PWD)/artifacts:/artifacts \
```
Comment (AlexeyRaga) [inline Makefile:10]: Too specific to certificates?

File: pack/entrypoint.sh [hunk 2]
```diff
@@ -61,18 +58,33 @@ dotnet new classlib --name $PACKAGE_NAME --output $SRC/$PACKAGE_NAME --framework
...
+
+for file in $(find $PROTOBUF_FOLDER -name "*.proto" -exec readlink -f {} \;)
+do
+  echo "Protobuf file found at '$file'. Trying to add it to the project..."
+  dotnet-grpc add-file  --services None --project $PROJ  $file
+done
+
+added_files=$(dotnet-grpc list --project $PROJ | grep -c "$PROTOBUF_FOLDER" )
+if [ $added_files -ne $count ]; then
+  echo "Error: The number of the Protobuf files found at the given path and the number of added files to the project doesn't march (found=$count,added=$added_files)!"
```
Comment (AlexeyRaga) [inline pack/entrypoint.sh:76]: Why would that ever happen?

**Review by AlexeyRaga — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (ali-tayebi) [reply Makefile:10]: This is only for test on dev time

File: pack/entrypoint.sh (hunk 2 above)
Comment (ali-tayebi) [reply pack/entrypoint.sh:76]: Just an approval process to ensure nothing left behind

**Pull request merged**